course_gather/management/commands/custom_db_sync.py

```diff
@@ -0,0 +1,125 @@
+from django.core.management.base import BaseCommand
+from course_gather.models import (
+    College,
+    Course,
+    MTUCourse,
+    State
+)
+from pathlib import Path
+import csv
+
+
+class Command(BaseCommand):
```

Comment:
Command is a reserved keyword to Django. This class _must_ be called Command.
BaseCommand is the object pass-in similar to the traditional `object` pass in.

The way this would be launched is using the `manage.py` file by:
`manage.py custom_db_sync /path/to/csv/file`